fda-model/metrics/ProjectPerformanceResults.cs

```diff
@@ -73,7 +86,56 @@ public double AssuranceOfAEP(double exceedanceProbability)
 
         public double ConditionalNonExceedanceProbability(double exceedanceProbability)
         {
-            double conditionalNonExceedanceProbability = _cnep[exceedanceProbability].CDF(_thresholdValue);
+            if (_calculatePerformanceForLevee)
+            {
+                return CalculateConditionalNonExceedanceProbabilityForLevee(exceedanceProbability);
+            }
+            else
+            {
+                double conditionalNonExceedanceProbability = _cnep[exceedanceProbability].CDF(_thresholdValue);
+                return conditionalNonExceedanceProbability;
+            }
+
+        }
+
+        private double CalculateConditionalNonExceedanceProbabilityForLevee(double exceedanceProbability)
```

Comment:
this is alot to digest, i am happy you have it working and passing tests, but it seems more complicated than i expected. I am thankful you figured it out!

Reply:
I am very open to simplifying if you have ideas. 

I basically followed Bob's write up on this.

Reply:
that is perfect - if we are passing tests, we can change it any time we want/need to for simplification as long as the test continue to pass.